.github/workflows/ci.yml

```
@@ -0,0 +1,74 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Set up Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python-version }}
+      - name: Install nox
+        run: |
+          python -m pip install --disable-pip-version-check --upgrade --user pip setuptools
+          python -m pip install .
+      - name: Run tests (ubuntu and macos)
```

What about showing the actual os running?

```suggestion
      - name: Run tests on ${{ matrix.os }}
```

---

Good idea, just applied that change :+1: